### DISCUSSION TITLE
Ability to run configure commands from ansible.

### DISCUSSION THREAD

**Discussion by natebeck72** [PaloAltoNetworks/pan-os-ansible#242] [Ideas]
I have some use cases where I would need to be able to run configure level commands, such as creating device groups, or TLS profiles etc.  I looked through all the modules and currently the use cases I am searching for do not exist, if need be I can compile a list of the things that I am searching for if that might be on the roadmap.

Thanks

**Comment by shinmog** [comment]
You can run arbitrary configuration commands using [panos_type_cmd](https://paloaltonetworks.github.io/pan-os-ansible/modules/panos_type_cmd.html)

Creating device groups was added in the latest release (2.8.0) just FYI
